CHANGELOG.adoc

```diff
@@ -17,9 +17,14 @@
 |===
 | | Description | PR
 
+| 🎁
+| Add `--context` and `--cluster` flags to choose specific context from kubeconfig
```

Comment:
I couldn't decide if that shouldn't be split to 2 entries per flag. Please let me know wdyt.

Reply:
I think this is fine